src/jvm.jl

```diff
@@ -200,6 +201,7 @@ function init(opts)
     jvm = unsafe_load(pjvm)
     global jvmfunc = unsafe_load(jvm.JNIInvokeInterface_)
     global jnifunc = unsafe_load(jnienv.JNINativeInterface_) #The JNI Function table
+    initProxy()
```

Comment:
We should revert the changes to this file for now.

Reply:
Looks like init_current_vm() needs initProxy() as well